data/hails.txt

```diff
@@ -1098,6 +1172,11 @@ phrase "friendly navy"
 		"the biggest"
 		"the largest"
 		"the greatest"
+		"the strongest"
+		"the grandest"
+		"the righteous"
```

Comment:
"The righteous" doesn't work here. It would have to be "the most righteous."

Reply:
Right, cause not a holy brigade. was just thinking of large ways to describe an entire navy, in their eyes they'd be righteous, no?

Reply:
You misunderstand me. The string in question could lead to wrong-sounding output like this:

"Join the Navy, the righteous defense force in the galaxy!"